xbmc/windowing/gbm/DRMUtils.h

```diff
@@ -30,34 +30,34 @@
 
 struct plane
 {
-  drmModePlane *plane;
-  drmModeObjectProperties *props;
-  drmModePropertyRes **props_info;
+  drmModePlane *plane = nullptr;
```

Comment:
Is it ok to initialize structs this way?

Reply:
because we use c++11 in kodi its fine, yes.